app/models/case_group.rb

```diff
@@ -1,6 +1,6 @@
 class CaseGroup < ApplicationRecord
   belongs_to :casa_org
-  has_many :case_group_memberships
+  has_many :case_group_memberships, dependent: :destroy
```

Comment:
This fixes a foreign key constraint issue when clicking delete on the Case Groups page.